apigee_sync.go

```diff
@@ -235,6 +249,14 @@ func downloadDataSnapshot() {
 	scopes = append(scopes, apidInfo.ClusterID)
 	resp := downloadSnapshot(scopes)
 
+	knownTables = extractTablesFromSnapshot(resp)
+
+	db, err := data.DBVersion(resp.SnapshotInfo)
+	if err != nil {
+		log.Panicf("Database inaccessible: %v", err)
```

Comment:
I think Scott's right, this code is only called in the main routine, not in the goroutine for polling changes, which I believe is the only other goroutine instantiated.